src/plugins/kernel_install/systemd_bootloader.sh

```diff
@@ -0,0 +1,45 @@
+SYSTEMD_BOOT_ENTRIES_DIR='/boot/loader/entries'
+
+function run_bootloader_update()
+{
+  local flag="$1"
+  local target="$2"
+  local name="$3"
+  local sudo_cmd
+  local loader_conf="${SYSTEMD_BOOT_ENTRIES_DIR}/kw_${name}_linux.conf"
+
+  if [[ -e "/boot/vmlinuz-${name}" ]]; then
+    add_systemd_boot_entry "$flag" "$target" "$name"
+  fi
+}
+
+function add_systemd_boot_entry()
+{
+  local flag="$1"
+  local target="$2"
+  local name="$3"
+  local loader_conf_path="${SYSTEMD_BOOT_ENTRIES_DIR}/kw_${name}.conf"
+  local template_loader_path="${REMOTE_KW_DEPLOY}/template_loader.conf"
+  local kernel_parameters
+  local cmd
+  local sudo_cmd
+
+  [[ "$target" == 'local' ]] && sudo_cmd='sudo'
+
+  # Get kernel parameters from default entry
+  #TODO: only use jq to do the parsing
+  kernel_parameters=$(bootctl --json=short list | jq '.options' | head -n1 | tr -d '"')
```

Comment:
This command outputs `jq: error (at <stdin>:1): Cannot index array with string "options"`, which fails to fill the `options` field in the bootloader entry and prevents a successful kernel boot.
If we're interested in obtaining the information from the first entry, I suggest:
```suggestion
  kernel_parameters=$(bootctl --json=short list | jq -r '.[0].options')
```